package.json

```diff
@@ -1,6 +1,26 @@
 {
   "name": "homeday-blocks",
   "version": "2.5.0",
+  "author": "Homeday - Frontend Team",
+  "description": "A Vue component library built by Homeday's frontend team.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/homeday-de/homeday-blocks.git"
+  },
+  "keywords": [
+    "vue.js",
+    "vueJS",
+    "Javascript",
+    "Component library",
+    "storbybook",
+    "homeday-block",
+    "services"
+  ],
+  "bugs": {
+    "url": "https://github.com/homeday-de/homeday-blocks/issues"
+  },
+  "homepage": "https://github.com/homeday-de/homeday-blocks#readme",
```

Comment:
What about using `https://blocks.homeday.dev/` for the homepage? We have a quite nice welcome page now and it also links to the GitHub page. Since we will have a dedicated github link on npm anyway, might be more valuable to use this one for something else.

Reply:
Was thinking about that as well. Agree, will update. :+1:

Reply:
I'm with @OriginalEXE here as well.

Reply:
Done. :+1: